monero-outreach-docs/translations/ru/transcriptions/breaking_monero/13_blockchain_explorer_ru.md

```diff
@@ -0,0 +1,18 @@
реров. В седьмом эпизоде мы уже говорили об удалённых узлах и некоторых моментах, которые необходимо учитывать при их использовании. Но многие забывают о своём сетевом соединении, когда пользуются блокчейн-эксплорерами. Поэтому, когда люди хотят посмотреть на транзакцию, которая была отправлена ими или подтверждена остальной частью сети, они часто не думают о том, каковы могут быть последствия их действий. Итак, сегодня с вами я, Джастин, и Саранг, и мы вкратце напомним вам о тех сетевых ограничениях, с которыми вы сталкиваетесь, когда взаимодействуете с блокчейн-эксплорером. Я поделюсь с вами одним снимком, чтобы продемонстрировать некоторые из этих эксплореров. Возьмём, к примеру, xmrchain.net, являющийся одним из наиболее часто используемых блокчейн-эксплореров Monero. Выглядит он, может, и не так привлекательно, но на нём вы найдёте всю необходимую информацию. Допустим, вы захотите найти информацию о пяти ваших транзакциях, проведёте поиск, чтобы найти их. Шансы довольно высоки, так как я ищу информацию о транзакции, с которой непосредственно связан. В данном случае, конечно, это была буквально первая из появившихся, и я никак не связан с ней, но если вы продолжите и сделаете всё сами, особенно, если вы занимались именно поиском транзакций, поиском пяти ваших собственных транзакций за определённый промежуток времени, этот веб-сайт довольно хорошо сможет установить связь между ними. Но для начала веб-сайт получает мой IP-адрес, и веб-сайту также известно, что конкретно с этого IP-адреса человек искал эти транзакции, совершённые за этот временной период. Таким образом, у них, как ни у кого другого в сети, имеется представление о моей связи с этими выходами. Подобным образом существуют прекрасные сайты сбора и предварительной обработки данных для xmrchain.net, такие как ExploreMonero.com, при помощи которых вы с лёгкостью можете верифицировать свои транзакции. То есть вы можете сказать: «Я отправил Monero», - и ввести сюда подробную информацию, и это действительно удобно и хорошо с этой точки зрения, но просто нужно понять, что если вы говорите о том, что получили Monero, и даёте подробную информацию о таком получении, то этот веб-сайт будут иметь очень сильные основания, чтобы подозревать вашу связь с транзакцией, по которой вами был сделан запрос. Он сможет связать данные с вашим IP и с тем IP-адресом, с которого вы будете делать запрос несколько раз, допустим, о пяти транзакциях, и высока вероятность, что ExploreMonero предположит, что вы связаны со всеми этими пятью транзакциями. Это просто напоминание о том, что когда вы используете блокчейн-эксплореры, вы раскрываете большой объём информации, и многие люди, в спешке пытающиеся получить информацию о своих транзакциях, часто забывают об этом в процессе. Я передаю слово Сарангу, который расскажет, что делать, если вы хотите получить информацию из блокчейн-эксплорера, но не хотите раскрывать свою информацию, или как использовать блокчейн-эксплорер, чтобы лучше защитить свои сетевые метаданные.
```

Comment:
- [remote nodes in episode 7](https://www.monerooutreach.org/breaking-monero/remote-nodes.php)
- [xmrchain.net](https://xmrchain.net/)
- [ExploreMonero.com](https://www.exploremonero.com/)